docs/reference/setup/secure-settings.asciidoc

```diff
@@ -16,6 +16,10 @@ Elasticsearch.
 NOTE: The elasticsearch keystore currently only provides obfuscation. In the future,
 password protection will be added.
 
+These settings, just like the ones in the regular `elasticsearch.yml` config file,
+need to be specified for each cluster node. These are node-specific settings,
```

Comment:
I'd suggest maybe changing to "... need to be specified on each node in the cluster."